main/templates/main/navbar.html

```diff
@@ -1,7 +1,15 @@
 <!--Navbar-->
+{% load static %}
 <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
   <div class="container">
-    <a class="navbar-brand" href="{% url 'main:index' %}">DUNE Run Control</a>
+    <a class="navbar-brand" href="{% url 'main:index' %}">
+      <img src="{% static 'main/images/DUNElogo_white.png' %}"
+           alt="DUNE Run Control"
+           width="80"
+           height="50"
+           class="d-inline-block align-top">
+      <h2 class="d-inline-block">Run Control</h2>
```

Comment:
Maybe following this example more closely will align the text better?

https://getbootstrap.com/docs/5.0/components/navbar/#image-and-text

I guess that's maybe something like this (could also involve changing width and height):
```suggestion
           class="d-inline-block align-text-top">
      Run Control
```